Doc/library/subprocess.rst

```diff
@@ -567,6 +567,13 @@ functions.
       Popen destructor now emits a :exc:`ResourceWarning` warning if the child
       process is still running.
 
+   .. versionchanged:: 3.8
+      Popen can now use :func:`os.posix_spawn` in some cases for better
+      performance. On some very specific platforms, Popen constructor using
```

Comment:
Can you mention some example of that platforms?

Reply:
I prefer to not mention WSL and QEMU User Emulation since it's a bug and these bugs can be fixed whenever. I prefer to not mention them to be more future-proof (not have to update the doc).

Reply:
I'd mention the platforms anyways with a "at the time of this writing, buggy platforms we are aware of include Windows Subsystem for Linux (version XYZ) and QEMU (version XYZ)" ...  If there are bugreports filed against them, those can be linked to from the doc as well.

Otherwise it such text sets off alarm bells for anyone reading it, they'll have the same question: am I impacted? do I use any "very specific platforms"? and come seeking more information.

when we become aware of upstream bugs being fixed we can update the text to say "versions prior to ABC" or similar.